### PR TITLE
Adds various elm-css helpers

### PR DESCRIPTION
**Pull request by scottcorgan**
Adds
- cursor
- overflowWrap
- wordWrap
- zIndex
- content

### CODE REVIEW

File: src/Nri/Css/Extra.elm [hunk 1]
```diff
@@ -14,3 +37,115 @@ import Css exposing (..)
...
+content : String -> Mixin
+content value =
+    property "content" <| "'" ++ value ++ "'"
+
+
+{-| z-index helper
+-}
+zIndex : Int -> Mixin
+zIndex value =
+    property "z-index" <| toString value
```
Comment (eeue56) [inline src/Nri/Css/Extra.elm:44]: why not just make these against elm-css?

File: src/Nri/Css/Extra.elm [hunk 2]
```diff
@@ -14,3 +37,115 @@ import Css exposing (..)
 lineHeightNum : Float -> Mixin
 lineHeightNum height =
     property "line-height" <| toString height
+
+
+{-| Helper for content CSS attribute
+-}
+content : String -> Mixin
+content value =
+    property "content" <| "'" ++ value ++ "'"
```
Comment (eeue56) [inline src/Nri/Css/Extra.elm:37]: why not just make this against elm-css?

File: src/Nri/Css/Extra.elm [hunk 3]
```diff
@@ -14,3 +37,115 @@ import Css exposing (..)
...
+    | NotAllowed
+    | Default
+    | Auto
+
+
+{-| Css cursor helper
+-}
+cursor : Cursor -> Mixin
+cursor value =
+    property "cursor" <| cursorToString value
```
Comment (eeue56) [inline src/Nri/Css/Extra.elm]: this already exists in elm-css http://package.elm-lang.org/packages/rtfeldman/elm-css/6.1.0/Css#cursor

Comment (eeue56) [reply src/Nri/Css/Extra.elm]: http://package.elm-lang.org/packages/rtfeldman/elm-css/6.1.0/Css#cursor

File: src/Nri/Css/Extra.elm (hunk 1 above)
Comment (scottcorgan) [reply src/Nri/Css/Extra.elm:44]: I was doing that, but features/bugs are released in batches (2 weeks to a month at a time). I just wanted to have a place to throw these until the PRs get merged.

File: src/Nri/Css/Extra.elm (hunk 3 above)
Comment (scottcorgan) [reply src/Nri/Css/Extra.elm]: didn't see that before, thanks!

**Comment by eeue56** [comment]
I'm still more in favour of pushing this upstream rather than locally

**Comment by scottcorgan** [comment]
Fair enough

**Pull request closed without merging**